docker-hub/publish/insights-analytics.md

```diff
@@ -47,84 +48,73 @@ The HTTP API endpoints are available at:
+| ----------------------------- | ------------------------------------------------------------------------------------------------------------ | ---------------- |
+| Action                        | Request type, see [Action classification rules][1]. One of `pull_by_tag`, `pull_by_digest`, `version_check`. | January 1, 2022  |
+| Action day                    | The date part of the timestamp: `YYYY-MM-DD`                                                                 | January 1, 2022  |
+| Country                       | Request origin country.                                                                                      | January 1, 2022  |
+| Digest                        | Image digest.                                                                                                | January 1, 2022  |
+| HTTP method                   | HTTP method used in the request, see [registry API documentation][2] for details.                            | January 1, 2022  |
+| Host                          | The cloud service provider used in an event.                                                                 | January 1, 2022  |
+| Namespace                     | Docker [organization][3] (image namespace).                                                                  | January 1, 2022  |
+| Reference                     | Image digest or tag used in the request.                                                                     | January 1, 2022  |
+| Repository                    | Docker [repository][4] (image name).                                                                         | January 1, 2022  |
+| Tag (included when available) | Tag name. Only available if the request referred to a tag.                                                   | January 1, 2022  |
+| Timestamp                     | Date and time of the request: `YYYY-MM-DD 00:00:00`                                                          | January 1, 2022  |
+| Type                          | The industry from which the event originates. One of `business`, `isp`, `hosting`, `education`, `null`       | January 1, 2022  |
+| User agent tool               | The application a user used to pull an image (for example, `docker` or `containerd`).                        | January 1, 2022  |
+| User agent version            | The version of the application used to pull an image.                                                        | January 1, 2022  |
+| Domain                        | Request origin domain, see [Privacy][5].                                                                     | October 11, 2022 |
+
+[1]: #action-classification-rules
+[2]: /registry/spec/api/
+[3]: /docker-hub/orgs/
+[4]: /docker-hub/repos/
+[5]: #privacy
 
 ### Summary data
 
 The summary data format contains the following data points for each namespace,
 repository, and reference (tag or digest), for the selected time span.
```

Comment:
At some point in the near future, we will have different levels of summary data (for instance, repo-level summaries as well as tag/digest-level summaries), so we will need to figure out how best to document that. Just a heads-up going forwards.